softdev.bib

```diff
@@ -3333,3 +3333,10 @@ @article{yanovski21ghostcell
   year={2021},
   month = Aug,
 }
+
+@misc{cheng20typefreezing,
+    title = "Type freezing: exploiting attribute type monomorphism in tracing JIT compilers",
```

Comment:
You'll need to put curly braces around "JIT" or it'll be lower-cased.

Reply:
added :point_right: b6f38e5d98b984a25b021e9e64a98b5c8a568c88